### ISSUE TITLE
conllr R package

### ISSUE THREAD

**Issue by jwijffels** [nikopartanen/conllu-vis#1]
Hello, I wonder if that conllr R package is available somewhere?

**Comment by nikopartanen** [comment]
Hi! I didn't add it into GitHub as it contained quite messy functions which are now bit all over the place, but I added it now to repository [nikopartanen/conllr](https://github.com/nikopartanen/conllr), I hope you find it interesting. [Here](https://nikopartanen.github.io/conllr/conllr.html) is one example of how I have been using it. There is now so much JavaScript and other stuff there that I'm not sure how it works as an R package, but just clone it and try to compile `conllr.Rmd`, for example. 

I can't guarantee that anything there works as I haven't looked into it for some months, but my basic idea is that I want to be able to read CONLL-U files into R as a dataframe, modify that, and write it back into CONLL-U file, ideally without losing sentence id's or comments. I didn't come up with very satisfactory solution with the writing part though. It seems that since the last spring the situation with other packages has also improved a lot, so I have to check it again. 

Besides reading and writing it would be nice to be able to embed UD example sentences into an RMarkdown document. I think this would probably need setting up a new HTML widget of some sort.  I had no time to look more into it, but I think this would be very useful anyway. 

The way I've now rendered it into RMarkdown is something like this:

    conllu <- read_conllu('file.conllu')
    conllu %>% 
          filter(id == 'relevant_example') %>%
          as_conllu_div()

So now it just formats the example as a div-element which conllu.js processes onward. The example I [linked](https://nikopartanen.github.io/conllr/conllr.html) above should show it better. But this demands quite much setup for the RMarkdown document to include all the CSS and JavaScript in the header, but it worked for the few cases I needed it for. There are tons of situation it doesn't take into account, but it is a proof of concept.

There also doesn't seem to be any simple way to retrieve individual examples from the UD corpora without reading in entire files. Ideally one could retrieve any item with the id and language name, for example. I wonder if there is an API or package for that.
